app/models/user.rb

```diff
@@ -1,16 +1,22 @@
 # frozen_string_literal: true
 require 'common/models/base'
+require 'mvi/messages/find_candidate_message'
+require 'mvi/service'
 
 class User < RedisStore
   NAMESPACE = REDIS_CONFIG['user_store']['namespace']
   REDIS_STORE = Redis::Namespace.new(NAMESPACE, redis: Redis.current)
   DEFAULT_TTL = REDIS_CONFIG['user_store']['each_ttl']
+  MVI_SERVICE = VetsAPI::Application.config.mvi_service
 
   # id.me attributes
   attribute :uuid
   attribute :email
   attribute :first_name
+  attribute :middle_name
   attribute :last_name
+  attribute :gender
+  attribute :dob, Common::UTCTime
```

Comment:
This value comes from the SAML Assertion attribute in the form: `YYYY-MM-DD`.  `Common::UTCTime` uses `Time.parse` which may trip up without additional config.  I think just a plan Date may be more appropriate
```ruby 
attribute :dob, Date
```